Vizzuality/half-earth-v3
# Fix/metadata translation

## Fix metadata translation
### Description
Metadata translation was not being fetched correctly.
Also, ad react-query to cache the fetches and use it instead of swr

### Testing instructions
Check the metadata for all terrestrial vertebrates layer on biodiversity and change language to spanish
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-483?atlOrigin=eyJpIjoiYWJhOTcwMTI5ODViNDg2Y2JmZWQyYWY1YTIwY2E4ZTUiLCJwIjoiaiJ9
